android/app/src/main/AndroidManifest.xml

```diff
@@ -44,42 +47,42 @@
-        <service
-            android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationListenerService"
-            android:exported="false" >
-            <intent-filter>
-                <action android:name="com.google.android.c2dm.intent.RECEIVE" />
-            </intent-filter>
-        </service>
+      <service android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationRegistrationService"/>
+      <service
+        android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationListenerService"
+        android:exported="false" >
+        <intent-filter>
+          <action android:name="com.google.android.c2dm.intent.RECEIVE" />
+        </intent-filter>
+      </service>
 
       <!-- background-location services (1 of 2) -->
-        <service android:name="com.transistorsoft.locationmanager.BackgroundGeolocationService" />
-        <service android:name="com.transistorsoft.locationmanager.LocationService" />
-        <service android:name="com.transistorsoft.locationmanager.ActivityRecognitionService" />
-        <service android:name="com.transistorsoft.locationmanager.geofence.GeofenceService" />
-        <service android:name="com.transistorsoft.locationmanager.scheduler.ScheduleService" />
-        <service android:name="com.transistorsoft.locationmanager.scheduler.ScheduleAlarmService" />
-        <receiver android:enabled="true" android:exported="false" android:name="com.transistorsoft.rnbackgroundgeolocation.BootReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.BOOT_COMPLETED" />
-            </intent-filter>
-        </receiver>
+      <service android:name="com.transistorsoft.locationmanager.BackgroundGeolocationService" />
+      <service android:name="com.transistorsoft.locationmanager.LocationService" />
+      <service android:name="com.transistorsoft.locationmanager.ActivityRecognitionService" />
+      <service android:name="com.transistorsoft.locationmanager.geofence.GeofenceService" />
+      <service android:name="com.transistorsoft.locationmanager.scheduler.ScheduleService" />
+      <service android:name="com.transistorsoft.locationmanager.scheduler.ScheduleAlarmService" />
+      <receiver android:enabled="true" android:exported="false" android:name="com.transistorsoft.rnbackgroundgeolocation.BootReceiver">
+        <intent-filter>
+          <action android:name="android.intent.action.BOOT_COMPLETED" />
+        </intent-filter>
+      </receiver>
 
-        <!-- Google Maps API key -->
-        <meta-data
-          android:name="com.google.android.geo.API_KEY"
-          android:value="AIzaSyD3tJy2Eb_gp1KSxGsVKN-BY1XgytmHQTY" />
+      <!-- Google Maps API key -->
+      <meta-data
+        android:name="com.google.android.geo.API_KEY"
+        android:value="AIzaSyD3tJy2Eb_gp1KSxGsVKN-BY1XgytmHQTY" />
```

Comment:
I will not accept embed keys into version control.  Please see comment on PR #141.